README.rst

```diff
@@ -670,6 +670,15 @@ the Google homepage. Scripts can cause additional network requests to be
 performed, as well as modify the page or its layout, causing another round of
 page rendering and painting.
 
+---------
+
+\* Well... almost anything. We do need to have some scope here, else we're going to end up talking about quantum mechanics, or how human sight works. To that effect our boundries are:
```

Comment:
"boundaries"

also you probably should stick to 80 characters like most of the document